include/deal.II/grid/grid_generator.h

```diff
@@ -506,8 +506,8 @@ namespace GridGenerator
    * computed adaptively such that the resulting elements have the least
    * aspect ratio.
    *
-   * If colorize is set to true, the inner, outer, left, and right boundary
-   * get indicator 0, 1, 2, and 3, respectively. Otherwise all indicators are
+   * If colorize is set to true, the inner, outer and equatorial boundary
+   * get indicator 0, 1, and 2, respectively. Otherwise all indicators are
```

Comment:
I'm not sure what you mean by "equatorial". Do you mean the "radial" boundaries? (That may not be clearer either -- maybe more words are necessary.)